app/plugins/monitoring/README.rst

```diff
@@ -0,0 +1,32 @@
+==================================
+CrateDB Admin UI Monitoring Plugin
+==================================
+
+The CrateDB admin UI monitoring plugin allows you to monitor the average number
+and speed of ``SELECT``, ``INSERT``, ``UPDATE``, and
+``DELETE`` queries.
+
+Installation
+============
+
+For this plugin to work properly, make sure the ``/conf/plugins.json`` file
+contains the following::
+
+	{
```

Comment:
afaik rst requires indentation for code blocks.

also, please specify the code block language

`.. code-block:: json`